docs/core_modules/model_modules/embeddings/usage_pattern.md

```diff
@@ -56,23 +56,22 @@ To configure the model used (from Hugging Face hub), add the model name separate
 from llama_index import ServiceContext
 
 service_context = ServiceContext.from_defaults(
-  embed_model="local:sentence-transformers/all-mpnet-base-v2"
+  embed_model="local:BAAI/bge-large-en"
 )
 ```
 
 ### Embedding Model Integrations
 
-We also support any embeddings offered by Langchain [here](https://python.langchain.com/docs/modules/data_connection/text_embedding/), using our `LangchainEmbedding` wrapper class.
+We also support any embeddings offered by Langchain [here](https://python.langchain.com/docs/modules/data_connection/text_embedding/).
 
 The example below loads a model from Hugging Face, using Langchain's embedding class.
 
 ```python
-from langchain.embeddings.huggingface import HuggingFaceEmbeddings
-from llama_index import LangchainEmbedding, ServiceContext
+from langchain.embeddings.huggingface import HuggingFaceBgeEmbeddings
```

Comment:
nit: misspelling

Reply:
Hmm, rhe class name is `HuggingFaceBgeEmbeddings`

Reply:
https://github.com/langchain-ai/langchain/blob/c0acbdca1b5884ac90d17908fb2bb555a9ed9909/libs/langchain/langchain/embeddings/huggingface.py#L179

Reply:
oh - then the sentence below needs to be changed i think? `embed_model = HuggingFaceEmbeddings(model_name="BAAI/bge-base-en")`